src/services/kratos/cron.ts

```diff
@@ -0,0 +1,34 @@
+import { kratosAdmin, listSessionsInternal } from "./private"
+
+import { listIdentities } from "."
```

Comment:
possible circular reference, please check

Reply:
will check. is there a way to automatically detect those references?

Reply:
fixed in 664a70632ec7c5cd00c54fafe559477bdf004c80